### PR TITLE
data:Sandeep277

### PR DESCRIPTION
**Pull request by Sandeep277**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4520"></a>

### CODE REVIEW

**Comment by Tejender1521** [comment]
Everything looks good and ready to be merged.

File: data/Sandeep277.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+{
+  "name": "Sandeep Kushwaha",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student",
+  "avatar": "https://github.com/Sandeep277.png",
```
Comment (baibhav-tiwari) [inline data/Sandeep277.json:7]: you can remove the avatar line,  it will directly fetch  your real time image from your github.

**Review by baibhav-tiwari — CHANGES_REQUESTED** [review]

File: data/Sandeep277.json (hunk 1 above)
Comment (Sandeep277) [reply data/Sandeep277.json]: now avatar line is removed 
thanks for your feedback

**Comment by baibhav-tiwari** [comment]
LGTM 🚀

**Review by priyankarpal — APPROVED** [review]
Awesome 🎉 Thanks for your first contribution.

**Review by iamthecloverly — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**